README.md

```diff
@@ -47,7 +47,7 @@ middleman server
 
 The preview server allows you to build your site, by modifying the contents of the `source` directory, and see your changes reflected in the browser at: `http://localhost:4567/`
 
-To get started, simply develop as you normally would by building HTML, CSS, and Javascript in the `source` directory. When you're ready to use more complex templates, simply add the templating engine's extension to the file and start writing in that format.
+To get started, simply develop as you normally would by building HTML, CSS, and JavaScript in the `source` directory. When you're ready to use more complex templates, simply add the templating engine's extension to the file and start writing in that format.
```

Comment:
`JavaScript` is a brand, so in general you shouldn't write it w/o capitalizing the `S` when you capitalize the `J`.

Reply:
:+1:

Reply:
:+1: